modules/social_features/social_event/modules/social_event_invite/config/install/views.view.event_manage_enrollment_invites.yml

```diff
@@ -28,7 +28,7 @@ display:
     position: 0
     display_options:
       access:
-        type: event_owner_or_organizer
+        type: none
```

Comment:
This is now handled by the `RouteSubscriber` and `Access` check together with other routes and blocks.